src/backend/backend.ts

```diff
@@ -178,3 +181,40 @@ export interface VMBackend {
    */
   noModalDialogs: boolean;
 }
+
+/**
+ * execOptions is options for VMExecutor.
+ */
+export type execOptions = childProcess.CommonOptions & {
+  /** Output encoding; defaults to utf16le. */
```

Comment:
This comment is true for WSL only, right? It's not set for lima, so does that mean it defaults to utf-8 ?

Reply:
Fixed in 0d89d352fc0d1745ada9606ca00967f364d59e5d.